src/Operators/remapping.jl

```diff
@@ -103,6 +93,66 @@ function overlap_weights(
     return kron(Y_ov, X_ov)
 end
 
+function overlap_weights(
```

Comment:
Just my little OCD speaking, but I would put the 1D version of this function before the 2D version above. Great work though! Looks great to me